modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java

```diff
@@ -200,7 +200,9 @@ public void register(Registry registry) {
 			"2.0.0", "2.0.1",
 			new com.liferay.dynamic.data.mapping.internal.upgrade.v2_0_1.
 				UpgradeAutocompleteDDMTextFieldSetting(
-					ddmFormJSONDeserializer, ddmFormSerializer));
+					ddmFormJSONDeserializer, ddmFormSerializer),
+			new com.liferay.dynamic.data.mapping.internal.upgrade.v2_0_1.
```

Comment:
@linolaoj, can you please clarify why did you choose this Schema version to add the Upgrade Step?

Reply:
I choose a version that would allow the upgrade step to run for 7.2 and master, and also not to conflict with changes that are made on form validation by steps with a higher version.